src/Jabbr-Core.Data/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
@adamtuliper should I change anything in the .Data project? Or let @ashanhol change after we merge?